topics/transcriptomics/tutorials/differential-isoform-expression/slides.html

```diff
@@ -0,0 +1,257 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Identification of non-canonical ORFs and their potential biological function"
+contributors:
+  - gallardoalba
+
+priority: 4
+
+video: no
+zenodo_link: ""
+
+
+---
+
+## Index of contents
+
+1. Introduction
+
+2. Galaxy workflow
+
+---
+
+## Introduction
+
+- What do we mean by non-canonical ORFs?
+
+- Why study non-canonical ORFs?
+
+- Why have not been yet characterized?
+
+- How identify non-canonical ORFs?
+
+---
+
+## What do we mean by non-canonical ORFs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/cambridge_dictionary.png %})
```

Comment:
could you add better alt text here please?

Reply:
Sure!